diceRoll/diceRollUnitTests.js

```diff
@@ -140,6 +140,52 @@ function runTests(runTF) {
         return !(subRandomIntForDice(testGlobals.sumArray_expanded).includes('d'));
       }),
     ];
+    console.log('DISPLAY FUNCTION TESTS:');
+    passFail += [
+      new UnitTest('toggleSaved()', function() {
+        var result;
+        if (g.contentStatus === content.calculator) {
```

Comment:
In a test, you shjould setup the state of the app to a known a state, then test it. You shjould have a test that sets the g.contentStatus to cointent.calculator, then tests toggleSaved, and then another test that sets g.contentStatus to savedMenu, then test that. Your tests should be called:

'toggleSaved() from calculator'
'toggleSaved() from savedMenu

Your tests also shouldn't preferably not change the state of the app for other tests ... You should pass your state object into the test, or reset the state at the end. You can have code that runs before and after every test to reset the state.